Therapeut Vechter/Assets/Scripts/MainMenuScript.cs

```diff
@@ -0,0 +1,37 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+using UnityEngine.Audio;
+
+public class MainMenuScript : MonoBehaviour
+{
+    public AudioMixer Mixer;
+    public GameObject[] Menus;
+    public string ChangeScene;
+    public string[] VolumeName;
+
+    void Update()
+    {
+        if (Input.GetKeyDown(KeyCode.Escape)&& Menus[1].activeInHierarchy == false)

```

Comment:
This seems very hardcodes, if doing it like this an array is bad, would be better to define 3 menus with different names instead of an array